test/test_library/functional/test_library.py

```diff
@@ -734,109 +677,50 @@ def test_s3(self):
 
     def test_files(self):
 
-        l = self.get_library()
-
-        l.purge()
+        lib = self.get_library()
 
-        # fls = l.files
+        lib.purge()
 
         for e in [(str(i), str(j)) for i in range(10) for j in range(3)]:
-            f = l.files.new_file(path='path' + e[0], ref="{}-{}".format(*e),
-                                 source_url = 'foo', group=e[1], type_=e[1])
-
-            f = l.files.new_file(path='path'+e[0], ref="{}-{}".format(*e),
-                                 source_url = 'foo', group=e[1], type_=e[1])
+            lib.files.new_file(
+                path='path' + e[0], ref='{}-{}'.format(*e),
+                source_url='foo', group=e[1], type_=e[1])
 
-        self.assertEquals(30, len(l.files.query.all))
+        self.assertEquals(30, len(lib.files.query.all))
 
         # Will throw an exception on duplicate error
-        f1 = l.files.new_file(path='ref-a', type = 'type-a', source = 'source-a', state = 'a')
-        self.assertEquals('a', l.files.query.path('ref-a').one.state)
+        lib.files.new_file(path='ref-a', type='type-a', source='source-a', state='a')
+        self.assertEquals('a', lib.files.query.path('ref-a').one.state)
 
         # Test that it overwrites inistead of duplicates
-        f2 = l.files.new_file(path='ref-a', type='type-a', source='source-a', state = 'b')
-        self.assertEquals('b', l.files.query.path('ref-a').one.state)
-        self.assertEquals(31, len(l.files.query.all))
+        lib.files.new_file(path='ref-a', type='type-a', source='source-a', state='b')
+        self.assertEquals('b', lib.files.query.path('ref-a').one.state)
+        self.assertEquals(31, len(lib.files.query.all))
 
+    @unittest.skip('Are codes removed from test bundle?')
```

Comment:
There should be a test bundle that does have codes in example.com, but they may not be in the testbundle. The test code should probably be modified to select which bundle is used. The main value of the testbundle is that it can be saved right after being built and returned to pristine with an rsync, which speeds up tests a lot. We probably don't need to do that for all tests.